src/Countries/US/Indiana/HamiltonIncome/V20200101/HamiltonIncome.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\Indiana\HamiltonIncome\V20200101;
+
+use Appleton\Taxes\Countries\US\Indiana\HamiltonIncome\HamiltonIncome as BaseHamiltonIncome;
+
+class HamiltonIncome extends BaseHamiltonIncome
+{
+    public function getTaxRate(): float
+    {
+        return 0.0011;
```

Comment:
this is .11%?

Reply:
:exit: too bad the middle schools are closed, still haven't gotten that math lesson